eslintrc.js

```diff
@@ -0,0 +1,199 @@
+        "@typescript-eslint/type-annotation-spacing": "off",
+        "@typescript-eslint/unbound-method": "error",
+        "@typescript-eslint/unified-signatures": "error",
+        "arrow-body-style": ["error", "always"],
+        "arrow-parens": ["off", "as-needed"],
+        "class-methods-use-this": "off",
+        complexity: "error",
+        "constructor-super": "error",
+        curly: "error",
+        "default-case": "error",
+        "dot-notation": "error",
+        "eol-last": "off",
+        eqeqeq: ["error", "always"],
+        "guard-for-in": "error",
+        "linebreak-style": "off",
+        "max-classes-per-file": ["error", 1],
+        "max-len": [
+            "error",
+            {
+                code: 140,
+            },
+        ],
+        "max-lines": ["error", 1000],
+        "new-parens": "off",
+        "newline-per-chained-call": "off",
+        "no-bitwise": "error",
+        "no-caller": "error",
+        "no-cond-assign": "error",
+        "no-console": "error",
+        "no-debugger": "error",
+        "no-duplicate-case": "error",
+        "no-duplicate-imports": "error",
+        "no-empty": "off",
+        "no-empty-functions": "off",
+        "no-eval": "error",
+        "no-extra-bind": "error",
+        "no-extra-semi": "off",
+        "no-irregular-whitespace": "off",
+        "no-magic-numbers": [
+            "error",
+            {
+                ignore: [-1, 0, 1, 2, 3],
+            },
+        ],
+        "no-multiple-empty-lines": "off",
+        "no-new-func": "error",
+        "no-new-wrappers": "error",
+        "no-plusplus": [
+            "error",
+            {
+                allowForLoopAfterthoughts: true,
+            },
+        ],
+        "no-restricted-syntax": ["off", "ForInStatement"],
+        "no-return-await": "error",
+        "no-sequences": "error",
+        "no-sparse-arrays": "error",
+        "no-template-curly-in-string": "error",
+        "no-throw-literal": "error",
+        "no-trailing-spaces": "off",
+        "no-undef-init": "error",
+        "no-unsafe-finally": "error",
+        "no-unused-labels": "error",
+        "no-useless-constructor": "error",
+        "no-var": "error",
+        "no-void": "error",
+        "object-shorthand": "error",
+        "one-var": "error",
+        "padding-line-between-statements": [
+            "off",
+            "error",
+            {
+                blankLine: "always",
+                prev: "*",
+                next: "return",
+            },
+        ],
+        "prefer-arrow/prefer-arrow-functions": "error",
+        "prefer-const": "error",
+        "prefer-object-spread": "error",
+        "prefer-readonly": "error",
+        "prefer-template": "error",
+        "quote-props": "off",
+        radix: "error",
+        "space-before-function-paren": "off",
+        "use-isnan": "error",
+        yoda: "error",
+        "@typescript-eslint/tslint/config": [
```

Comment:
Amusing that there are still so many not yet implemented. You can go ahead and not include this list of `tslint/config` rules. I'm ok with them not being enabled any more. 😮